.gitignore

```diff
@@ -1,3 +1,7 @@
+# Smashbox - ignore pyco client and local config
+/src/
```

Comment:
Leading slash is to ignore it only in the root of the repo.
So if we add it somewhere else one day, it still works in a different folder